### DISCUSSION TITLE
/cn/2020/10/text-similarity/

### DISCUSSION THREAD

**Discussion by leovan** [leovan/leovan.me#19] [Comments]
文本相似度 (Text Similarity) - Leo Van | 范叶亮 
 https://leovan.me/cn/2020/10/text-similarity/

**Comment by leovan** [comment]
```header
由 gitopen 于 2022-01-14 09:39:52 写道：
````
作者总结的太好了。

**Comment by haleylgc** [comment]
哥 余弦相似度也是文本距离度量方法的一种

**Reply by leovan** [reply]
文本做向量表示后，就能使用很多其他相似度了，上文中「表示学习」部分有提及。
